nevergrad/benchmark/test_experiments.py

```diff
@@ -36,14 +36,18 @@ def test_experiments_registry(name: str, maker: tp.Callable[[], tp.Iterator[expe
     if all(x in name for x in ["image", "quality"]) and platform.system() == "Windows":
         raise SkipTest("Image quality not guaranteed on Windows.")
 
+    # ANM does not work under Windows.
+    if "gym" in name and platform.system() == "Windows":
+        raise SkipTest("Some gym problems suffer on Windows.")
```

Comment:
If windows is the problem, raise ng.errors.UnsupportedExperiment("Windows is not supported") directly in the experiment, this will not be useful anymore

Reply:
Done.